app/javascript/packages/document-capture/components/acuant-selfie-capture-canvas.jsx

```diff
@@ -21,6 +21,12 @@ function AcuantSelfieCaptureCanvas({ imageCaptureText, onSelfieCaptureClosed })
   // The Acuant SDK script AcuantPassiveLiveness attaches to whatever element has
   // this id. It then uses that element as the root for the full screen selfie capture
   const acuantCaptureContainerId = 'acuant-face-capture-container';
+
+  const elementInShadow = document
```

Comment:
How does this update if Acuant _becomes_ loaded? Are we expecting it to?

Reply:
Here's what I think is happening:
1. `<AcuantSelfieCamera>` is created, and Acuant is "started", but crucially the capture window doesn't appear anywhere yet.
1. `<FullScreen>` activates a focus trap around `<AcuantSelfieCaptureCanvas>`.
1. At this moment there's nothing inside that focus trap except a button and a div because Acuant hasn't actually attached to the div with `acuantCaptureContainerId`. 
1. Then, Acuant attaches to the div with `acuantCaptureContainerId` and hydrates in their Shadow DOM.


The problem that this PR solves is that it lets me show the button on Step 3, and hide it on Step 4. Practically, what happens if you try to fully remove the button is you get an error about the FocusTrap always needing a tabbable element (which makes sense).
- So that close button needs to be present until the moment that Acuant's Shadow DOM gets attached.
- Having that button seems to be the cause of some odd behavior when using a screenreader so it's worthwhile to remove it.

Reply:
The primary concern I have here is that I don't see where we predictably expect React to re-render this component after the changes you describe.

Reply:
A good point, the re-render isn't necessarily tied to this code in the way I want. We do re-render the `AcuantSelfieCaptureCanvas` every time the help text changes (frequently) so this value does get updated.